java-client/src/test/java/org/opensearch/client/opensearch/_types/query_dsl/XyShapeFieldQueryTest.java

```diff
@@ -0,0 +1,14 @@
+package org.opensearch.client.opensearch._types.query_dsl;
```

Comment:
This one needs a license header too (only SPDX for the new file, but this one does look like a copy of the old ShapeFieldQueryTest. CI should be catching the missing one - if it's not and it's not a trivial fix, open a GH issue please?

Reply:
Ok I've checked the other files in test directory before pushing, they don't have license header ? 
It is indeed a copy of ShapeFieldQueryTest, the header wasn't there before, is that normal ?
I am not sure to understand

Reply:
Sorry if I wasn't clear.

1. It should have been there and CI should have caught this as a problem before.
2. Please do add it, including the ES one since it's an old file.
3. We need to make sure license checker catches this in CI, if the problem is not obvious (excluded folder or something), please open an issue?

Reply:
> It is indeed a copy of ShapeFieldQueryTest, the header wasn't there before, is that normal ?
> I am not sure to understand

@kmessaoudi oh this seems like a miss on our side, thank you for pointing it out, I believe we need to have license headers , @VachaShah may be it rings a bell to you?